docs/Advanced Features/workflows.md

```diff
@@ -0,0 +1,143 @@
+*New in Quilt 3.3*
+
+`quilt3` provides API to ensure quality of package metadata. To start use it you need to create
+configuration file in your registry under `.quilt/workflows/config.yml`. For example:
```

Comment:
Shouldn't we provide this as a constant from `quilt3`, or from an API?

Reply:
I think that currently there is no good place for it.

Reply:
What about util.py? See https://docs.quiltdata.com/api-reference/cli#constants-see-util-py-for-more